compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -1942,6 +1941,7 @@ object desugar {
     mayNeedSetter
   }
 
+  @unused
   private def derivedDefDef(original: Tree, named: NameTree, tpt: Tree, rhs: Tree, mods: Modifiers)(implicit src: SourceFile) =
```

Comment:
Actually remove it, if it's unused?

But perhaps this shouldn't be part of this PR regardless, should it?

Reply:
I can skip it. I noticed it not only because I'm literally `CheckUnused`, but I was chasing down code in `Desugar`.

Reply:
Sorry I forgot to remove the annotation; distracted by ambient conditions.